src/vue/common/EmailGetter.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <span
+    v-if="isLoaded"
+    class="email-getter"
+    :title="isTitled && result"
+  >
+    {{ result }}
+  </span>
+  <span
+    v-else-if="isLoadingFailed"
+    class="email-getter"
```

Comment:
You can use `template` to avoid duplicating the `<span class="email-getter">` part:

```
<span class="email-getter">
  <template v-if>Content</template>
  <template v-else>Another content</template>
</span>
```